pkg/eosclient/eosgrpc/eosgrpc.go

```diff
@@ -16,6 +16,11 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
 
+// NOTE: compile the grpc proto with these commands
+// and do not ask any questions, I don't have the answer
+// protoc ./Rpc.proto --go-grpc_out=.
```

Comment:
we can add these commands in the makefile

Reply:
Actually this is not to be executed at every build, rather it stays "frozen" until we sync with eos